src/clj/rems/css/styles.clj

```diff
@@ -381,11 +389,11 @@
                     :min-height (u/px 300)
                     :max-width content-width
                     :flex-grow 1
-                    ;; Height of navigation + logo, to avoid page content going under
+                    ;; Height of navigation, to avoid page content going under
                     ;; the navigation bar when the main content is focused.
                     ;; See https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors
-                    :padding-top (u/px 212)
-                    :margin-top (u/px -212)}]
+                    :padding-top recalculated-menu-height
+                    :margin-top (u/px (- (int (+ menu-height 6))))}]
```

Comment:
This does not anymore take into account the hero logo being there so can this work?

Reply:
Yes, this is true, let me re-add it

Reply:
actually, now that i remember, min-height was actually meant for this part

Reply:
What `min-height`?

Reply:
should be back at this point